worker/introspection/script.go

```diff
@@ -127,12 +129,11 @@ juju_stop_unit () {
     echo "usage: juju_stop_unit <unit-name> [<unit-name>...]"
     return 1
   fi
-  arr=("$@")
-  local -a args
-  for i in "${arr[@]}"; do
-    args+=("unit=$i")
+  args=""
```

Comment:
Minor nit; this loop should probably be extracted in a separate func and reused below.

Reply:
Thanks for pointing that out @achilleasa 

Actually, the whole juju_start_unit/juju_stop_unit could have been abstracted into a _juju_action_unit but I didn't want to change the code that much for a small compatibility fix.

Thanks for the merge !